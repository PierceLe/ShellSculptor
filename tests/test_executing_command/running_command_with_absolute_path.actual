>> ./dont_have_this_file.sh
mysh: command not found: ./dont_have_this_file.sh
>> cd ~/test_data/
cd: no such file or directory: /Users/piercele/test_data/
>> ./file_without_permision.sh
mysh: command not found: ./file_without_permision.sh
>> ./file_with_permision.sh
mysh: command not found: ./file_with_permision.sh

