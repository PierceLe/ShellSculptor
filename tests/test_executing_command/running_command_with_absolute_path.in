./dont_have_this_file.sh
cd ~/test_data/
./file_without_permision.sh
./file_with_permision.sh
