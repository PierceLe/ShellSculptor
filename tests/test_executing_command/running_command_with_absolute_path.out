>> ./dont_have_this_file.sh
mysh: command not found: ./dont_have_this_file.sh
>> cd ~/test_data/
>> ./file_without_permision.sh
mysh: permission denied: ./file_without_permision.sh
>> ./file_with_permision.sh
You have permision to access this file haha

